src/functions/assertions/HaveParameter.ps1

```diff
@@ -264,14 +264,16 @@
         }
 
         if ($Alias) {
-            $testPresenceOfAlias = $ActualValue.Parameters[$ParameterName].Aliases -contains $Alias
-            $filters += "to$(if ($Negate) {" not"}) have an alias '$Alias'"
+            foreach ($AliasValue in $Alias) {
+                $testPresenceOfAlias = $ActualValue.Parameters[$ParameterName].Aliases -contains $AliasValue
+                $filters += "to$(if ($Negate) {" not"}) have an alias '$AliasValue'"
```

Comment:
```suggestion
            $filters += "with$(if ($Negate) {'out'}) alias$(if ($Alias.Count -ge 2) {'es'}) $(Join-And ($Alias -replace '^|$', "'"))"
            foreach ($AliasValue in $Alias) {
                $testPresenceOfAlias = $ActualValue.Parameters[$ParameterName].Aliases -contains $AliasValue
```

Should we the filter-text outside foreach and to improve readability?
We can utilize helpers in `Format.ps1` like `Join-And` or `Format-Collection` to make it pretty.

**Current PR:** 

> Expected command Invoke-DummyFunction to have a parameter MandatoryParam, to have an alias 'First' and to have an alias 'Second', but it didn't have an alias 'Second'.

**Join-And (suggestion):** 

> Expected command Invoke-DummyFunction to have a parameter MandatoryParam, with aliases 'First' and 'Second', but it didn't have an alias 'Second'.

**Format-Collection:** 

> Expected command Invoke-DummyFunction to have a parameter MandatoryParam, with aliases @('First', 'Second'), but it didn't have an alias 'Second'.

`$filters += "with$(if ($Negate) {'out'}) alias$(if ($Alias.Count -ge 2) {'es'}) $(Format-Collection $Alias)"`

Reply:
Thanks for the suggestions. They help to make the output more readable. I applied them, including to the formatting of the wrong aliases.